api_app/db/repositories/airlock_requests.py

```diff
@@ -31,18 +31,18 @@ def get_resource_base_spec_params():
     def get_timestamp(self) -> float:
         return datetime.utcnow().timestamp()
 
-    def update_airlock_request_item(self, original_request: AirlockRequest, new_request: AirlockRequest, user: User, request_properties: dict) -> AirlockRequest:
+    def update_airlock_request_item(self, original_request: AirlockRequest, new_request: AirlockRequest, updated_by: User, request_properties: dict) -> AirlockRequest:
         history_item = AirlockRequestHistoryItem(
             resourceVersion=original_request.resourceVersion,
             updatedWhen=original_request.updatedWhen,
-            user=original_request.user,
+            updatedBy=original_request.updatedBy,
```

Comment:
as per other comment - looks like the first time this creates the History Item, the updatedBy will be empty? unless i've missed something.

Reply:
Fixed, thanks!